src/day-1-toy/printf.py

```diff
@@ -2,9 +2,12 @@
 y = 2.24552
 z = "I like turtles!"
 
-# Using the printf operator (%), print the following feeding in the values of x,
+# Using the printf operator (%), print the following feeding in the values of    x,
 # y, and z:
-# x is 10, y is 2.25, z is "I like turtles!"
+print("Integer: %d, Float: %f, String: %s" %(x,y,z))
 
+# x is 10, y is 2.25, z is "I like turtles!"
+# print()
 
-# Use the 'format' string method to print the same thing
\ No newline at end of file
+# Use the 'format' string method to print the same thing
+print('x is {0}, y is {1}, z is {2}'.format(x, y, z))
```

Comment:
The `{}` can be empty, and they'll be filled up by the values supplied to `format()`. _With that said_, I love this a lot. It's easier to keep track what is going in what.

Also, I'm a fan of f-strings. Are you?